pkg/scheduler/apis/config/v1alpha1/conversion.go

```diff
@@ -66,3 +66,39 @@ func Convert_config_KubeSchedulerLeaderElectionConfiguration_To_v1alpha1_KubeSch
 	out.LockObjectName = in.ResourceName
 	return nil
 }
+
+func Convert_v1alpha1_Plugins_To_config_Plugins(in *v1alpha1.Plugins, out *config.Plugins, s conversion.Scope) error {
+	if err := autoConvert_v1alpha1_Plugins_To_config_Plugins(in, out, s); err != nil {
+		return err
+	}
+
+	if in.PostFilter != nil {
+		postFilter, preScore := &in.PostFilter, &out.PreScore
+		*preScore = new(config.PluginSet)
+		if err := Convert_v1alpha1_PluginSet_To_config_PluginSet(*postFilter, *preScore, s); err != nil {
+			return err
+		}
+	} else {
+		out.PreScore = nil
```

Comment:
isn't this unnecessary?

Reply:
I'm not sure. I copy the logic from the auto generated conversions.

Reply:
It seems a little  redundant. I can remove it.

Reply:
It's fine... it kind of makes sense if you are converting into a dirty struct.